platform/features/imagery/src/controllers/ImageryController.js

```diff
@@ -141,8 +122,8 @@ define(
         ImageryController.prototype.paused = function (state) {
             if (arguments.length > 0 && state !== this.isPaused) {
                 this.isPaused = state;
-                // Switch to latest image
-                this.updateValues();
+                this.updateValues(this.nextValue);
```

Comment:
Ah, if you pause and unpause without a new value coming in, this.nextValue is undefined and it clears the displayed image.  Easy to fix:

```javascript
if (this.nextValue) { 
    this.updateValues(this.nextValue); 
    delete this.nextValue; 
}
```